lib/src/configuration.dart

```diff
@@ -605,6 +605,10 @@ class ClientResetError extends SyncError {
     return "SyncError message: $message category: $category code: $code isFatal: $isFatal";
   }
 
+  /// Initiates the client reset process.
+  ///
+  /// On Windows, all Realm instances for that path must be disposed before this method is called or an
```

Comment:
```suggestion
  /// All Realm instances for that path must be closed before this method is called or an
```

We should not be explicit about Windows here - generally, you should not expect things to work normally if you have an open Realm instance while you're deleting the file.